packages/dds/tree/src/index.ts

```diff
@@ -15,9 +15,21 @@ export {
 } from "./core/index.js";
 
 export type {
+	/**
+	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
+	 */
 	Listeners,
+	/**
+	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
+	 */
 	IsListener,
+	/**
+	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
+	 */
 	Listenable,
+	/**
+	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
+	 */
 	Off,
 } from "@fluidframework/core-interfaces";
```

Comment:
Looks like this is actually insufficient. While tsc supports this pattern, api-extactor and eslint do not. Notice that there is no API report change here.

Reply:
The same applies to the previous approach—there are no API Extractor updates for those changes either. But it shows the usual strikethrough in VSCode to indicate these types are deprecated.

Reply:
One approach is to copy these types into the tree, mark them as deprecated, and export them from there. This will update the api-extractor reports to reflect the deprecation status of these types.